RULES.md

```diff
@@ -2181,17 +2182,53 @@ A route's color and `route_text_color` should be contrasting.
 
 </details>
 
-<a name="RouteShortAndLongNameEqualNotice"/>
+<a name="RouteLongNameConainsShortName"/>
 
-### route_short_and_long_name_equal
+### route_long_name_contains_short_name
 
-A single route has the same values for `route_short_name` and `route_long_name`.
+Long name can not contain short name for a single route.
 
 Example of bad data:
```

Comment:
Would it be helpful to add the reason(s) why the following examples are considered 'bad data'?